src/java/picard/metrics/GcBiasMetrics.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2009 The Broad Institute
```

Comment:
Probably best to update it to 2015.  I don't know if the copyright wording has updated since 2009.  Maybe search the code base for a more recent copyright?